app/build.gradle

```diff
@@ -20,14 +22,40 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    dataBinding {
+        enabled = true
+    }
+}
+ext {
+    supportLibraryVersion = '28.0.0-beta01'
+    retrofitVersion = '2.4.0'
+    kotlin_version = '1.2.60'
+    junitVersion = '4.12'
+    picassoVersion = '2.71828'
+    lifecycleVersion = '1.1.1'
 }
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlin_version"
-    implementation 'com.android.support:appcompat-v7:28.0.0-beta01'
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    //noinspection GradleCompatible
+    implementation "com.android.support:appcompat-v7:$supportLibraryVersion"
     implementation 'com.android.support.constraint:constraint-layout:1.1.2'
-    testImplementation 'junit:junit:4.12'
+    testImplementation "junit:junit:$junitVersion"
     androidTestImplementation 'com.android.support.test:runner:1.0.2'
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
+    kapt 'com.android.databinding:compiler:3.1.3'
+    implementation "com.squareup.retrofit2:retrofit:$retrofitVersion"
+    implementation "com.squareup.retrofit2:converter-gson:$retrofitVersion"
+    implementation "com.squareup.retrofit2:adapter-rxjava2:$retrofitVersion"
+    implementation "com.android.support:recyclerview-v7:$supportLibraryVersion"
+    implementation'io.reactivex.rxjava2:rxjava:2.1.1'
+    implementation'io.reactivex.rxjava2:rxandroid:2.0.1'
+    implementation "com.squareup.picasso:picasso:$picassoVersion"
+    implementation "com.android.support:cardview-v7:$supportLibraryVersion"
+    implementation "android.arch.lifecycle:extensions:$lifecycleVersion"
+    implementation "android.arch.lifecycle:viewmodel:$lifecycleVersion"
```

Comment:
I think you don’t need the dependency for lifecycle:viewmodel since the viewmodel is already included in a lifecycle:extensions package